heron/healthmgr/src/java/BUILDX

```diff
@@ -0,0 +1,65 @@
+licenses(["notice"])
```

Comment:
The Dhalion build files and the dependencies have been temporarily disabled till maven repo is enabled. I would suggest we keep it like this for a few more days. Some CI systems are based on local maven repo only and cannot download jar files from the internet. The build will be enabled once maven repo is updated. Also, a couple of PRs related to Dhalion will follow this request. We split the PR in parts to simplify review.